build.sbt

```diff
@@ -785,7 +786,8 @@ lazy val standaloneUtil = (project in engine("standalone/util")).
 lazy val standaloneApi = (project in engine("standalone/api")).
   settings(commonSettings).
   settings(
-    name := "nussknacker-standalone-api"
+    name := "nussknacker-standalone-api",
+    libraryDependencies += "com.github.erosb" % "everit-json-schema" % everitSchemaV
```

Comment:
is it used ?

Reply:
Yes - `JsonSchemaStandaloneSourceFactory`

Reply:
It is used to compose schema validator and validate json from request

Reply:
Wouldn't it be easier to just use swagger-model?

Reply:
You mean to replace `everit` with `swagger-model`?